### PR TITLE
[FIX] Updated `structure_labels` function to output softmax probability instead of logit for the best category

### PR DESCRIPTION
**Pull request by wonjininfo**
[FIX] Updated `structure_labels` function to output softmax probability instead of logit for the best category

- Added `simple_softmax` function in `cnlp_predict.py` to calculate softmax values for a 1-D score array.
- Updated `structure_labels` function to use `simple_softmax` for probability. The TSV output now contains softmax probabilities instead of logits for the best category.

### CODE REVIEW

**Comment by wonjininfo** [comment]
I choose not to change the output format and instead just change the value to softmax probability.

We can change the output format to list all the logits or softmax probabilities for all the label categories, but since this will change the output format, I thought it would be better to do so after discussion or review.

**Review by tmills — APPROVED** [review]
